xds/internal/balancer/clusterresolver/priority_test.go

```diff
@@ -274,7 +279,8 @@ func (s) TestEDSPriority_SwitchPriority(t *testing.T) {
 	}
 
 	// Should get an update with 1's old picker, to override 2's old picker.
-	if err := testErrPickerFromCh(cc.NewPickerCh, balancer.ErrTransientFailure); err != nil {
+	want := errors.New("last connection error: subConn connection error")
+	if err := testErrPickerFromCh(cc.NewPickerCh, want); err != nil {
```

Comment:
I was about to make the same comment as I did on the other file for here, but realized that `testErrPickerFromCh` performs a `reflect.DeepEqual()` on the got and want errors, instead of doing a `strings.Contains()`. Would you mind fixing the implementation of `testErrPickerFromCh` to do the latter instead. It doesn't have to be done in this PR. You can send another PR for that after this is merged. Thanks.